testing/tests/DevExpress.ui.widgets.dataGrid/gridView.tests.js

```diff
@@ -9,15 +9,26 @@ QUnit.testStart(function() {
         height: 440px;\
     }\
 </style>\
-<div style="padding: 0px 40px; margin: 0px 50px">\
+<div id="testWrapper">\
     <div id="testContainer"></div>\
 </div>\
 <div id="root">\
     <div id="container" class="dx-datagrid dx-widget"></div>\
 </div>\
-<div id="itemsContainer"><div style="width:125px; display: inline-block;" ></div><div style="width:125px; display: inline-block;" ></div></div>';
+<div id="itemsContainer"><div></div><div></div></div>';
 
     $('#qunit-fixture').html(markup);
+
+    $('#testWrapper').css({
+        padding: '0 40px',
+        margin: '0 50px'
+    });
+
+    $('#itemsContainer > div').css({
+        width: '125px',
+        display: 'inline-block'
+    });
```

Comment:
Let's move this styles to ```<style>```:

```suggestion
    #testWrapper {
        padding: 0 40px;
        margin: 0 50px;
    }

    .itemsContainer__child {
        width: 125px;
        display: inline-block;
    }
```